ml-agents/mlagents/trainers/learn.py

```diff
@@ -349,7 +349,7 @@ def try_create_meta_curriculum(
         return None
 
     else:
-        meta_curriculum = MetaCurriculum(curriculum_folder)
+        meta_curriculum = MetaCurriculum.from_folder(curriculum_folder)
```

Comment:
nit: `from_directory`? "folder" seems Windows-y (although I guess a lot of the surrounding code uses it too).

Reply:
Sure, that sounds fine to me.